www/data/docs_sidebar.yml

```diff
@@ -28,6 +28,8 @@ sidebar_links:
     link: "/docs/reference/reporters.html"
   - title: InSpec DSL
     link: "/docs/reference/dsl_inspec.html"
+  - title: Style guide
```

Comment:
Consider labelling this the 'Profile Style Guide'.  We may wish to add a Resource Style Guide in the future, or a Ruby style guide.

Reply:
Missed that part, thank you!